modules/lightning_features/lightning_core/src/Access/AdministrativeRoleCheck.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\lightning_core\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\Routing\Route;
+
```

Comment:
I don't know; I'm following the pattern established in core. I'm not implementing an interface, though, so presumably I could remove it.